### ISSUE TITLE
Is IDEA IntelliJ 2024 supported?

### ISSUE THREAD

**Issue by maiq123** [jonathanlermitage/intellij-extra-toolwindow-colorful-icons#8]
Just wanted to confirm if IntelliJ's new 2024 update is supported. I updated my IntelliJ and all me icons are back to what it was before downloading this plugin.

**Comment by jonathanlermitage** [comment]
Yes it is. What's your exact IJ version (Help > About > Copy), and are you using the regular or the new UI? Any other 3rd-party plugin that could interfere with theming?

**Comment by maiq123** [comment]
All good, I had 2023.2.1 where it was working fine then I updated to 2023.2.6 and thought it was 2024.1. Updated to 2024.1 and its back. Although it wasn't working in 2023.2.6 though.
